plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -269,13 +283,27 @@ fn get_name_and_version(line: &str) -> (&str, &str) {
     (name, version)
 }
 
+fn get_config() -> Option<TedgeConfig> {
+    let config_dir = TEdgeConfigLocation::default();
+
+    match fs::read_to_string(config_dir.tedge_config_file_path()) {
+        Ok(content) => {
+            let config: Option<TedgeConfig> = toml::from_str(&content).unwrap_or_else(|e| {
+                eprintln!("Failed to parse TOML: {}", e);
```

Comment:
It would be good to tell which file cannot be parsed.

Reply:
It would be better to give the path actually used:

```
                eprintln!("Failed to parse {}: {}", config_dir.tedge_config_file_path(), e);
```